convert.c

```diff
@@ -256,6 +257,117 @@ static int will_convert_lf_to_crlf(size_t len, struct text_stat *stats,
 
 }
 
+static struct encoding {
+	const char *name;
+	iconv_t *to_git;
+	iconv_t *to_worktree;
+	struct encoding *next;
+} *encoding, **encoding_tail;
+static const char *default_encoding = "utf-8";
+
+static int encode_to_git(const char *path, const char *src, size_t src_len,
```

Comment:
Some short documentation on the meaning of the return values would be nice.

Reply:
`1` indicates successful encoding, `0` is indicates and error or no encoding... I guess that is the standard in this file as similar functions don't explicitly mention it.